web_programming/daily_horoscope.py

```diff
@@ -28,8 +28,18 @@ def horoscope(zodiac_sign: int, day: str) -> str:
         "11. Aquarius\n",
         "12. Pisces\n",
     )
-    zodiac_sign = int(input("number> ").strip())
-    print("choose some day:\n", "yesterday\n", "today\n", "tomorrow\n")
-    day = input("enter the day> ")
-    horoscope_text = horoscope(zodiac_sign, day)
-    print(horoscope_text)
+    zodiac_sign = int(input("Select a sign [1-12] > ").strip())
+    if zodiac_sign not in range(0, 13):
+        print("Invalid choice of Zodian sign!")
+    else:
+        days_list = {"1": "Yesterday", "2": "Today", "3": "Tomorrow"}
+        print("\nDays available:")
+        for day in days_list.items():
+            print("{}. {}".format(day[0], day[1]))
+        inp_day = input("Choose a day [1-3] > ")
+        if inp_day not in days_list:
+            print("Invalid choice of day!")
+        else:
+            day = days_list[inp_day].lower()
```

Comment:
Please condense these last 3 lines down to one line.

Reply:
I have made it one liner, still had to add an if condition to make horoscope function call.